.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
```

Comment:
> I was just trying to point out that as this is a low-frequency repo and Travis won't be run regularly and thus PRs can end up with unrelated failures.

I've just enabled the daily build cron job at travis, so bugs in dmd-nightly won't be able hide in the dark anymore ;)